lib/getPostcssResult.js

```diff
@@ -58,7 +58,7 @@ module.exports = function(stylelint /*: stylelint$internalApi*/) /*: Promise<?Ob
 						stringify: postcss.stringify,
 					};
 				}
-			} else if (syntax) {
+			} else if (stylelint._options.syntax) {
```

Comment:
Why not use `syntax` as before?

Reply:
here is a little bit tricky place `stylelint._options.syntax` is `undefined | string` so after assigning on line 41 `syntax` implicitly have type `undefined | string`, but after we assign to `syntax` object - `{parse: any, stringify: any}` as well.